<?xml version="1.0" ?>

<launch>

    <node pkg="ocs2_legged_robot_ros" type="legged_robot_gait_command" name="legged_robot_gait_command"
              output="screen"/>

    <node pkg="qm_controllers" type="qm_target_trajectories_publisher" name="qm_target_trajectories_publisher"
          output="screen"/>

</launch>
